gfortran_utils.sh

```diff
@@ -97,6 +97,21 @@ if [ "$(uname)" == "Darwin" ]; then
         hdiutil attach -mountpoint /Volumes/gfortran $GFORTRAN_DMG
         sudo installer -pkg /Volumes/gfortran/gfortran.pkg -target /
         check_gfortran
+        if [[ "${PLAT:-}" == "universal2" || "${PLAT:-}" == "arm64" ]]; then
```

Comment:
Just checking - do we also need the `gfortran.pkg` above?

Do we need to add an archive for this compiler as well?

Reply:
> Just checking - do we also need the gfortran.pkg above?

For some rare cases where for cross compiling you need a native compiler to generate some executables that are used during the build.

> Do we need to add an archive for this compiler as well?

I don't understand. Can you clarify?

Reply:
Or are the releases stored somewhere in version control - within Github?

Reply:
As you can see - I stored the binary blob for the installer in version control here to be sure we had it, and we knew exactly what it was.  The question is how to do the same for your gcc toolchain.  Can we refer to your repository commit or similar?  Are the binary artefacts guaranteed to be available for the medium term, for reproducible builds?

Reply:
Checked into this repo